plugins/woocommerce/tests/php/src/Internal/FraudProtection/CheckoutEventTrackerTest.php

```
@@ -516,4 +516,97 @@ function ( $event_data ) use ( $order ) {
+	public function test_track_pay_for_order_includes_required_fields(): void {
+		$order = \WC_Helper_Order::create_order();
+		$order->set_status( 'pending' );
+		$order->set_payment_method( 'bacs' );
+		$order->save();
+
+		$captured_event_data = null;
+		$this->mock_dispatcher
+			->expects( $this->once() )
+			->method( 'dispatch_event' )
+			->willReturnCallback(
+				function ( $event_type, $event_data ) use ( &$captured_event_data ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$event_type'. (undefined)

(UnusedFormalParameter)